### DISCUSSION TITLE
Trader NFTs

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1231] [Art] [answered]
We will be looking to launch some trader NFTs that give boosts for those trading such as increase daily listings, limits etc.

The first one we are after is a "Rich trader" NPC character. There have been some great ideas around Discord about what this character will look like. Some suggestions are a chunky character with a top hat or monocle. As always there are no clear outlines on what the item should be, let your imagination go wild!

**Requirements**

- Character has similar dimensions to other NPCs, beavers, roosters etc.
- Artwork is original (provide raw files)
- Submit the minified file (1px = 1px) and also a screenshot of it placed in the game.
- You agree to the game's terms and conditions for the art to be used.

**Deadline**

23rd of July

The winner will be airdropped their creation ❤️

**Comment by Kiulg** [comment]
How about a wealthy grumpy squirrel
### Hat ![sq](https://user-images.githubusercontent.com/18270876/179146901-4cfa06ff-f782-42e8-89a8-6ea63a977985.gif)

![111](https://user-images.githubusercontent.com/18270876/179146923-55b9c644-4597-4203-b9a6-757da574313a.gif)



### Without hat ![Squirrelwh](https://user-images.githubusercontent.com/18270876/179059228-a9c98fa7-a602-46ad-8f2f-df423692b60d.gif)

![222](https://user-images.githubusercontent.com/18270876/179147317-cd8f8021-94d1-42c0-ae28-cb666348486f.gif)


### V2 ![Squirrel21](https://user-images.githubusercontent.com/18270876/179072257-c1214177-cf73-462f-a83e-58fdb9834e19.gif)

![333](https://user-images.githubusercontent.com/18270876/179147623-412e7cfb-74b3-4606-b250-4ac4103c1cc9.gif)

**Reply by adamhannigan** [reply]
This is awesome! ❤️ Could we also get a minified version and one without the hat?

It could be cool to do the NPC bobbing animation with this grumpy squirrel 🤗

**Reply by Kiulg** [reply]
Thank you :D I really prefer the hat version it has more magic for the purpose, I did with typical standby animation but I'm going to do a more consistent

**Comment by Vergelsxtn** [comment]
Rich trader man🌻
![prev](https://user-images.githubusercontent.com/99605039/178920874-10d4b425-691f-4755-b18c-1f39a69ce09a.gif)
Prev:
![richtrader-capprev](https://user-images.githubusercontent.com/99605039/178921186-99bf19ed-6b99-4198-8f28-e4a77f517ed8.gif)
![richtrader-nocapprev](https://user-images.githubusercontent.com/99605039/178921194-ac3e7eec-01de-422e-9bb5-e03de4f512ee.gif)
Assets:
![richtrader-cap](https://user-images.githubusercontent.com/99605039/178921197-4a221dc5-e5bc-47d1-8758-cafb4a41d310.gif)
![richtrader-nocap](https://user-images.githubusercontent.com/99605039/178921206-8cb4b9a5-00d3-49ed-9df0-1d30c8a65a60.gif)

**Comment by JcEii** [comment]
### Rich Koala and Panda Trader NFT

**Sample Images:**
![Sample](https://user-images.githubusercontent.com/105348036/178940865-62058fb3-edc8-497d-b936-be462173d67d.gif)
![koala normal](https://user-images.githubusercontent.com/105348036/178940905-c305a24d-397a-4ae1-a07a-eaae8c2e2ae6.gif)
![koala special](https://user-images.githubusercontent.com/105348036/178940907-53a4b2ed-6718-4fa1-a6e2-d3fd5da21c05.gif)
![panda normal](https://user-images.githubusercontent.com/105348036/178940912-443bef43-cb88-4ee1-9cb4-496272ccab0e.gif)
![panda speciall](https://user-images.githubusercontent.com/105348036/178940917-dbb62892-7627-4cc2-a51f-e64927903f3a.gif)

**Minified Assets:**
![koala normals](https://user-images.githubusercontent.com/105348036/178940969-ce51e2ff-e74e-4379-ba68-6df319e2dd7e.gif)
![koala special-small-](https://user-images.githubusercontent.com/105348036/178940974-a26bf044-fa1c-46e5-81f4-ce2a622a19e7.gif)
![panda normals](https://user-images.githubusercontent.com/105348036/178940978-b7601a60-b94b-4636-8571-451946add06d.gif)
![panda specials](https://user-images.githubusercontent.com/105348036/178940980-02eb07b9-cd85-44e9-b31b-5adbe06f4733.gif)


**Aseprite File:**
[NFT TRADER by jc eii.zip](https://github.com/sunflower-land/sunflower-land/files/9110141/NFT.TRADER.by.jc.eii.zip)


Discord: Jc Eii #3727
Farm no.: 15747

**Reply by TaRaSiSt** [reply]
Why did you post the same thing twice)?

**Reply by JcEii** [reply]
@TaRaSiSt The 2 npc's have their own items unlike the 2.

**Reply by JcEii** [reply]
Definition:
Senior Koalsky - Richest koalan trader in the world.
Mr. Panchy -  Richest Panda in Goblin Town.

**Comment by VadimasVP** [comment]
### Rich Goblin

### 1. ![rich_goblin2](https://user-images.githubusercontent.com/102822587/178952580-4f8f8138-88b3-445f-bc47-653b141f139b.gif)

![rich_goblin](https://user-images.githubusercontent.com/102822587/178952699-f2ee60d5-35c1-40e4-97ff-81128822af79.gif)
![rich_goblin1](https://user-images.githubusercontent.com/102822587/178952702-77dad45a-fff9-48f0-a8b2-043d15a3f110.gif)

### 2. ![rich_bumpkin32](https://user-images.githubusercontent.com/102822587/179347335-107ef739-badd-4910-a1b1-d82490304704.gif)
![rich_goblin54](https://user-images.githubusercontent.com/102822587/179347338-a94ce10e-3627-4683-85de-61700bff788f.gif)
![rich_goblin4](https://user-images.githubusercontent.com/102822587/179347336-d34389e9-7dca-45af-a684-ab571db339c8.gif)



### Rich Bumpkin

### 1. ![rich_bumpkin2](https://user-images.githubusercontent.com/102822587/179347154-2fc7eadf-cc82-4426-bc2e-7060b878d26d.gif)

![rich_bumpkin1](https://user-images.githubusercontent.com/102822587/179347153-8dc0cabe-b6c1-4692-bd82-7753b0e87627.gif)
![rich_bumpkin7](https://user-images.githubusercontent.com/102822587/179347155-5f94350f-b712-41dd-b388-f413b6ff443d.gif)

### 2. ![rich_bumpkin4](https://user-images.githubusercontent.com/102822587/179347182-a012ca3c-00e4-4a49-876a-38d443bfceca.gif)

![rich_bumpkin34](https://user-images.githubusercontent.com/102822587/179347185-4d68c1eb-48f4-4e8c-8435-71ea61f73a1f.gif)
![rich_bumpkin6](https://user-images.githubusercontent.com/102822587/179347184-f69b1865-7fb6-426b-bba6-25d3565200a3.gif)

**Comment by TaRaSiSt** [comment]
**Mr.Trader**
![Trader](https://user-images.githubusercontent.com/47809767/179430269-658d30d7-3942-4789-bca0-a14b8c877691.gif)
Duck
![Duck](https://user-images.githubusercontent.com/47809767/178972603-b5f6257f-2b9f-4759-bf53-3df80fc604f0.gif)

**Assets**
![Trader](https://user-images.githubusercontent.com/47809767/179430190-9481d79a-d774-4707-bfb0-369448ad916a.png) ![Duck](https://user-images.githubusercontent.com/47809767/178972634-0726f82b-0be8-49c0-bf2c-612ed3fc8d24.png)

Animation
![TraderAnim](https://user-images.githubusercontent.com/47809767/179430195-f8bc812f-d0b9-40c8-ac58-6ae9a28e635f.gif) ![TraderBackground](https://user-images.githubusercontent.com/47809767/180758295-45792dc1-6a90-4bdd-b4b4-e4c74b47b23d.gif) ![DuckAnim](https://user-images.githubusercontent.com/47809767/178972660-26caa211-960c-4d46-88b9-3fb225af07a1.gif)

Discord TaRaSiSt#7782 ^_^
Farm id 28877

**Comment by sfwhat** [comment]
#### **Rich Racoon**

_A distant cousin of Victoria the cat_


![Sprite-0002](https://user-images.githubusercontent.com/95937808/179078179-6466917c-2029-4958-85cc-7862ca2d4e24.gif)


Assets
Wo/Hat
![hatlessracoon-export-export](https://user-images.githubusercontent.com/95937808/179077853-3124889b-08b4-4381-b812-e05079dcf263.gif)


W/Hat
![racoonhat](https://user-images.githubusercontent.com/95937808/179077978-9d533b50-ef33-4f1d-ad3c-bfd1204baf45.gif)



SfWhat
#6127
Twitter @strugwhat

**Comment by CeZura25** [comment]
**Mr. Toad**
_A golden Bullfrog from faraway land_

Sample:

![Mr  Toad](https://user-images.githubusercontent.com/89783564/179017026-f064bb3b-fad9-45b0-8216-cc97d68b6df9.gif)

Minified:
Mr. Toad 1 ![bull frog 1](https://user-images.githubusercontent.com/89783564/179017735-8bc8d86c-8571-4c0c-80a3-44c25c3be498.gif)
Mr. Toad 2 ![bull frog 2](https://user-images.githubusercontent.com/89783564/179017748-8a83cb54-8cdf-4251-a41a-8f1ce1294c8f.gif)
Mr. Toad 3 ![bull frog](https://user-images.githubusercontent.com/89783564/179017768-43618769-f877-450d-91ce-dd108338a8a2.gif)

Aseprite:
[my bullfrog Aseprite.zip](https://github.com/sunflower-land/sunflower-land/files/9113046/my.bullfrog.Aseprite.zip)

Discord: Danyel#5697
Farm ID: #14917

**Comment by FarmerKuenster** [comment]
Sir. Richard Loot

![rich-trader-small](https://user-images.githubusercontent.com/104322749/179084889-3f7cffbe-2926-4c94-b6ab-9f74a9d17800.gif)
![rich-trader-large-export](https://user-images.githubusercontent.com/104322749/179085012-b90a640e-6a6f-47aa-868e-8bc5e415ecb1.gif)
![rich-trader-in-sfl](https://user-images.githubusercontent.com/104322749/179084937-4445aa4e-55d3-4cdb-9240-573b477ce91f.gif)

discord: farmerkunester #1626
farm id: 32391

**Comment by On1el** [comment]
Hi all, I want to share with you my trader Ruru.
He saved a little crab from death a long time ago as a child and they have been inseparable friends ever since.
It turned out that the crab has a talent to find and mine all kinds of resources, after which my character became incredibly rich. 
Moving on the crab he can transport a lot of resources and effortlessly travel where others would break their legs or even part with their lives. Another great feature of the crab is its claws, in the market they work perfectly as scales and that's why trader Ruru never makes mistakes
**x1**
![traderx1](https://user-images.githubusercontent.com/104461938/179212694-e27a2246-1b87-4ee9-ae89-4746df18a331.gif)
**x5**
![traderx2](https://user-images.githubusercontent.com/104461938/179212740-7df5a3bc-f355-4d67-87eb-a56a9580d173.gif)

added shadows
![ONIEL](https://user-images.githubusercontent.com/104461938/180416526-ca420707-0908-4489-85fe-8d6185b9caf7.gif)
![traderx2 1](https://user-images.githubusercontent.com/104461938/180416553-83683b83-cc50-43d5-be5e-887efdefea3a.gif)

stands still
x1 ![traderx3](https://user-images.githubusercontent.com/104461938/180616268-65811def-fde0-445e-9505-9419705cd9a0.gif)
x5 ![traderx3500](https://user-images.githubusercontent.com/104461938/180616287-89d9a3b7-ed55-4de6-8825-0da6c3f9e4f2.gif)
![traderx3fon](https://user-images.githubusercontent.com/104461938/180616295-18eb1554-528c-45f4-be25-b393783e35b9.gif)

**V3**
x1 ![traderxV5](https://user-images.githubusercontent.com/104461938/180617593-432b928d-3aef-434f-86fc-dd6ae208d5fa.gif)
x5 ![traderxV5500](https://user-images.githubusercontent.com/104461938/180617596-06e454dd-1658-408c-a6a6-eeb3aa78906e.gif)

![traderxV5500fon](https://user-images.githubusercontent.com/104461938/180617600-b43c4f56-84f4-4513-a0db-5bec171d26b4.gif)

**Comment by tworkyfree** [comment]
**Goblin Trader**
I tried to draw a very rich and at the same time greedy goblin trader)
![goblin trader](https://user-images.githubusercontent.com/104664729/179286954-f60773e3-27f6-4d40-8246-2470050a9b00.gif)
![goblin trader1](https://user-images.githubusercontent.com/104664729/179286959-32d883a6-94ec-4618-b16f-36d9002d1eac.gif)
![demo](https://user-images.githubusercontent.com/104664729/179286961-ac782a77-260e-4fca-81f7-f7743f189a8d.gif)

**Comment by esp123** [comment]
**Rich Trader** 

Mr. Red
![RedPouch](https://user-images.githubusercontent.com/74337946/180239624-26d959cb-0463-4ab1-b5b3-bf5b98f274cb.png)
![RedPouchs](https://user-images.githubusercontent.com/74337946/180239633-9f794584-f82a-4393-8118-258243cb577f.gif)
![RedPouch](https://user-images.githubusercontent.com/74337946/180238658-e6757135-8dc4-4c9d-8a6e-773487b25747.gif)

Made it quite unique with suit and monocle trend. (Suit requested by lord Zikou).

Variations:
![Goldi-packs](https://user-images.githubusercontent.com/74337946/179326885-a029487b-efbc-447b-84e2-dae728cd2517.png)
![gp1](https://user-images.githubusercontent.com/74337946/179326890-0320d554-47fa-48b5-961c-1bae4d6efbf0.png)
![gp (2)](https://user-images.githubusercontent.com/74337946/179326893-6de4fc44-5f17-4023-8650-077a9b7a8046.png)
![gp (3)](https://user-images.githubusercontent.com/74337946/179326895-4f8d5ca8-2bb1-48cf-9518-6d0f906ba3b7.png)

Discord: deeefault#0033
Farm #: 80399

**Comment by eArL2384** [comment]
hmmn fat trader, Oooooohhh a whale got eeem 

![External (10)](https://user-images.githubusercontent.com/104702521/179967696-3443f6bf-e114-473e-aac3-4d65857317b4.gif)

![External (9)](https://user-images.githubusercontent.com/104702521/179967745-f6cdc596-7787-4f33-a739-1bfe9a95c489.gif)


![External (7)](https://user-images.githubusercontent.com/104702521/179967805-b7f1865c-65bf-4601-971f-133d785d2e67.gif)

![External (8)](https://user-images.githubusercontent.com/104702521/179967861-659352c0-480c-4f6e-b33c-883a3e06a710.gif)

![External (11)](https://user-images.githubusercontent.com/104702521/179967907-27eef0b7-1838-4fd4-a24b-2e2fe7730e8a.gif)

![External (12)](https://user-images.githubusercontent.com/104702521/179967937-92417b46-7977-4170-aee2-9887917b748c.gif)

revisions!

![whaletraderwithtail (1)](https://user-images.githubusercontent.com/104702521/180176873-dfa05188-d1bb-4cce-88c4-208e781456cf.gif)

![whaletrader (1)](https://user-images.githubusercontent.com/104702521/180176920-bd6b2d00-e117-4c1d-b7b9-75080bf0244b.gif)

![whalenftrevisedwithtail](https://user-images.githubusercontent.com/104702521/180176966-77b39a44-bc0e-4eeb-8f69-b69589c769eb.gif)

![whalenftrevised](https://user-images.githubusercontent.com/104702521/180177009-0c4d18a6-a25d-4228-b7c0-e7621f5a5632.gif)

resized


![resizedwhaletail](https://user-images.githubusercontent.com/104702521/180178063-e388402e-0b99-4d8f-9938-aa4d72647b81.gif)

![redo](https://user-images.githubusercontent.com/104702521/180178405-b047b318-c91e-4a27-95be-53950e4d308a.gif)


Farm ID#434
Discord ID: Earl2384-Ankhknown#6898

**Reply by On1el** [reply]
Definitely need to make the animation smoother

**Reply by eArL2384** [reply]
thank you mate definitely took your view into consideration 😊.

**Comment by ItsNight** [comment]
![q-export](https://user-images.githubusercontent.com/43474459/180263564-587f7c73-4c70-4ad7-91ed-1b61f53bfaa6.gif)
![q-export](https://user-images.githubusercontent.com/43474459/180263576-ac84c49a-0ce1-42a9-b8a8-cc0c3e24d9d8.png)
![q](https://user-images.githubusercontent.com/43474459/180263588-220ab94e-8412-4ea6-b81a-123238abad5f.png)
![q](https://user-images.githubusercontent.com/43474459/180263642-3fd5dd2a-6256-4528-b58d-f12b477a7db6.gif)
![image](https://user-images.githubusercontent.com/43474459/180263697-74a803a0-9e99-4cad-bd3c-cf973aae0b56.png)

**Comment by On1el** [comment]
x1 ![Криминальное-чтиво](https://user-images.githubusercontent.com/104461938/180622654-c67ec82d-9222-4f61-a3d1-cf0eb841843c.gif)
x5 ![Криминальное-чтиво500](https://user-images.githubusercontent.com/104461938/180622658-f0ca98d8-6da4-4efd-8778-4a9fb6107034.gif)

![Криминальное-чтивоFON](https://user-images.githubusercontent.com/104461938/180622664-ffc5bd72-6a3e-401a-b181-063f509bba3d.gif)

**Comment by adamhannigan** [comment, marked as answer]
Well done everyone! Creative ideas all round 🤗 

We have a winner.... 🥁 

@VadimasVP Rich Bumpkin

A crowd favourite and a perfect fit for the game ❤️ We will be adding this NFT over the coming week.

We are also looking to add some more trader NFTs in the future and will look to use another piece of art in this discussion so keep your eyes peeled 👀
